src/main/java/org/matsim/stuttgart/Utils.java

```diff
@@ -31,6 +31,18 @@
         return result;
     }
 
+
+    public static List<PlanCalcScoreConfigGroup.ActivityParams> createActivityPatterns(String type, long minDurationInSeconds, long maxDurationInSeconds, long durationDifferenceInSeconds, double openingHour, double closingHour) {
+
+        List<PlanCalcScoreConfigGroup.ActivityParams> result = createActivityPatterns(type, minDurationInSeconds, maxDurationInSeconds, durationDifferenceInSeconds);
+        result.forEach(activityParams -> {
```

Comment:
The current agreement in the department is that a normal for loop is preferable to a forEach, since a for loop is more idiomatic java.

Reply:
changed the stream to for loop at this place